docs/UserGuide.md

```diff
@@ -11,117 +11,287 @@
+> Priority options are: `h` for High, `m` for Medium, `l` for Low
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add Meet John Doe -dt 26/09/2016 0900 to 26/09/2016 1030 -t CATCH UP`
+* `add Complete CS2103 Quiz -dt 23/09/2016 -p h, -t Quiz -t CS2103, -r 13 EVERY WEEK`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Reserving timeslots for a task : `rsv`
+Reserves one or more timeslot for a task<br>
+Format: `rsv <TASK> -dt <START_DATE/TIME> to <END_DATE/TIME> [, <START_DATE/TIME> to <END_DATE/TIME>, …]`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> Words in `UPPER_CASE` are the parameters. 
+>
+> Time is in a 24-hr format. More than one date/time can be added.
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
-and persons matching at least one keyword will be returned (i.e. `OR` search).
+Examples:
+* `rsv Meet John Doe -dt 26/09/2016 0900 to 1030, 28/09/2016 1000 to 1130`
 
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+#### Editing a reserved timeslot : `rsv -e`
+Renames a task with reserved time slots or Adds/Deletes a reserved timeslot for a task <br>
+Format: `rsv -e <INDEX> -n <TASK> -dta <START_DATE/TIME> to <END_DATE/TIME> -dtr <START_DATE/TIME> to <END_DATE/TIME>`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+> Words in `UPPER_CASE` are the parameters. 
+> 
+> Time is in 24-hr format. More than one date/time can be edited.
+>
+> Use -n to rename the task <br>
+> Use -dta to add a timeslot <br>
+> Use -dtr to remove a timeslot <br>
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Examples:
+* `rsv -e 2 -n Meet John Tan -dta 08/10/2016 1000 to 1200`
+* `rsv -e 1 -n -dta 09/10/2016 1100 to 1230 -dtr 05/10/2016 1000 to 1200`
 
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+#### Deleting a task with reserved timeslots : `rsv -d`
+Deletes a task with all its reserved time slots <br>
+Format: `rsv -d <INDEX>`
+
+> Deletes the task at the specific `<INDEX>`. 
+> The index refers to the index number shown in the task list.
+> The index **must be a positive integer** 1, 2, 3, ...
+
+Examples:
+* `rsv -d 5`
+
+#### Confirming a reserved timeslot : `confirm`
```

Comment:
Oh ! this is a interesting thing to have! but you might want to do something with the others tasks! like perhaps shift them to another list like pending list and highlight the user! 👍 this is a interesting feature to have!!! *we can discuss more on this the next tutorial :) *